scipy/signal/tests/test_windows.py

```diff
@@ -502,6 +502,10 @@ def test_basic(self, xp):
 class TestKaiserBesselDerived:
 
     def test_basic(self, xp):
+        # cover case `M < 1`
```

Comment:
@jacobtomlinson it happened :)

```
/home/runner/work/scipy/scipy/scipy/signal/tests/test_windows.py (no explicit encoding; utf-8 assumed)
... line 505, position 10: character ' ', code point U+00A0
```

```suggestion
        # cover case `M < 1`
```